README.md

```diff
@@ -980,6 +980,7 @@ Table of Contents
   * [fly.io](https://fly.io/) - Fly is a platform for applications that need to run globally. It runs your code close to users and scales compute in cities where your app is busiest. Write your code, package it into a Docker image, deploy it to Fly's platform and let that do all the work to keep your app snappy. Free for side projects, $10/mo of service credit that automatically applies to any paid service. And if you ran really small virtual machines, credits will go a long way.
   * [Divio](https://www.divio.com/) - A platform to manage cloud application deploying only using Docker. Available free subscription for development projects, Requires card and no custom domain support.
   * [Koyeb](https://www.koyeb.com) - Koyeb is a developer-friendly serverless platform to deploy apps globally. Seamlessly run Docker containers, web apps, and APIs with git-based deployment, native autoscaling, a global edge network, and built-in service mesh and discovery. Koyeb provides two nano services to run your apps with its forever-free tier and also sponsors open-source projects with free resources.
+  * [Klotho](https://klo.dev) - Klotho is a cloud-aware coding toolbox that enables changing architectures in minutes, swapping technologies seamlessly and building every cloud application with 4 simple capabilities compatible with the same tools and libraries you already use and love.
```

Comment:
"Sign up for closed beta" I dont think that's appropriate for this list.

Reply:
Aha, apologies, will close for now and recreate if we become a fit.